documentation/modules/auxiliary/gather/exchange_proxylogon_collector.md

```diff
@@ -0,0 +1,149 @@
+## Vulnerable Application
+
+  This vulnerability affects (Exchange 2013 Versions < 15.00.1497.012, Exchange 2016 CU18 < 15.01.2106.013,
+  Exchange 2016 CU19 < 15.01.2176.009, Exchange 2019 CU7 < 15.02.0721.013, Exchange 2019 CU8 < 15.02.0792.010).
+
+### Introduction
+
+  An issue was discovered in Microsoft Exchange Server that allows an attacker bypassing the authentication and
+  impersonating as the admin (CVE-2021-26855). By chaining this bug with another post-auth arbitrary-file-write
+  vulnerability to get code execution (CVE-2021-27065).
+
+  As a result, an unauthenticated attacker can execute arbitrary commands on Microsoft Exchange Server.
+
+  All components are vulnerable by default.
+
+## Verification Steps
+
+1. Start msfconsole
+2. Do: `use auxiliary/gather/exchange_proxylogon`
+3. Do: `set RHOSTS [IP]`
+4. Do: `set EMAIL [EMAIL ADDRESS]`
+5. Do: `set SERVER_NAME [SERVER_NAME]`
+6. Do: `run`
+
+## Options
+1. `EMAIL`. The email account what you want dump
+2. `FOLDER`. The email folder what you want dump. Default: `inbox`
+3. `Proxies`. This option is not set by default.
+4. `RPORT`. The default setting is `443`. To use: `set RPORT [PORT]`
+5. `SERVER_NAME`. The name of secondary internal Exchange server targeted.
+6. `SSL`. The default setting is `true`.
+7. `VHOST`. This option is not set by default.
+
+## Scenarios
```

Comment:
I think the documentation does not reflect the last additions made to the module: actions, contact dump and attachment download.